.eslintrc.yaml

```diff
@@ -15,19 +15,26 @@ globals:
   CryptoJS: true
   dcodeIO: true
   Dexie: true
+  entities: true
   hljs: true
+  jQuery: true
   keyboardJS: true
   ko: true
   libsodium: true
+  lorem_ipsum: true
```

Comment:
It's part of the payloads.js. We might want to move this, entities and whatever is in there to a dedicated namespace in a separate PR.